src/app/components/client/PageLoadEvent.tsx

```diff
@@ -56,9 +56,11 @@ export const PageLoadEvent = (props: Props) => {
     // record attributions on page load
     if (window.location.search?.length > 0) {
```

Comment:
I'm a bit worried about having this in the same `useEffect` as the page view recording - I'm not sure, but I think it might result in that effect running too often, resulting in over-counting page views. It might be safer to have both the first-touch and last-touch cookie set in their own separate effects?

Reply:
Hmm, good call. It might be possible that this func runs twice, once on page load, and once when `cookies` change.. I'm not sure either how likely this happens, but sure I'll separate it out into its own `useEffect`

Reply:
Changed in https://github.com/mozilla/blurts-server/pull/4047/commits/72f2d48adcbe0786da722d338d1b3db39e3a5db6